r/R/compute.R

```diff
@@ -286,3 +286,8 @@ cast_options <- function(safe = TRUE, ...) {
   )
   modifyList(opts, list(...))
 }
+
+strptime_arrow <- function(..., format, unit){
+  a <- collect_arrays_from_dots(list(...))
```

Comment:
I don't think you want `collect_arrays_from_dots` here. This function exists to support the base R behavior like:

```
> sum(1, 2)
[1] 3
```

But `strptime` doesn't take `...` like that.